resources/views/emails/company/ebe5-upgrades-reminder-de.blade.php

```diff
@@ -0,0 +1,51 @@
+
+
+@component('mail::message')
+
+# Hallo {{$profile->translate("[[fname]]")}},
+
+**Wir wollen dich letztmals an die Möglichkeit der Bestellung von zusätzlichen Dienstleistungen für die E-Commerce Berlin Expo aufmerksam machen.** 
+
+Du kannst alle Informationen diesbezüglich hier finden:  
+
+@component('mail::button', ['url' => $accountUrl])
+Zu meinem Account gelangen
+@endcomponent
+
+**Das Angebot gilt bis zum 15.01.2020**
```

Comment:
@peterecommberlin jak nie dajesz podwójnej nowej linii to zgodnie z markdown to sie polaczy do 1. przypuszczam ze tak nie chcesz.

Reply:
a gdzie są te zasady spisane jak przygotować plik? ten "jezyk" sie pewnie jakos nazywa